### DISCUSSION TITLE
Too many questions to test update to script

### DISCUSSION THREAD

**Discussion by carlin-q-scott** [microsoft/vscode-azureAutomation#47] [Ideas]
Great extension, but it could be a little easier to use.

I'd like to have a one-click test of updates to my runbook. Right now it takes 9+ clicks:

1. Right-click
2. Upload draft
3. Yes
4. Right-click
5. Publish
6. Yes
7. Right-click
8. Run Job
9. Yes
10. Enter for each parameter in script to use default

**Comment by Kapilhk** [comment]
Hi @carlin-q-scott , Thanks for bringing this up, Although the 1-4 are redundant coming to this extension as only 'Publish' command is sufficient to Upload and Publish.
Sure we will consider providing a combination command that will Publish as well as Run a Job.
